stable/redis-ha/values.yaml

```diff
@@ -70,7 +70,6 @@ persistentVolume:
 redis_image: gcr.io/google_containers/redis:v1
 
 
-
```

Comment:
Can you delete this line?

Reply:
removed. but circici linter complains about too many blank lines 😕